frontend/src/main/web/lib/stores/GlossaryStore.js

```diff
@@ -0,0 +1,356 @@
+import Dispatcher from '../dispatchers/GlossaryDispatcher';
+import assign from 'object-assign';
+import {EventEmitter} from 'events';
+import Configs from '../constants/Configs';
+import {GlossaryActionTypes} from '../constants/ActionTypes';
+import GlossaryHelper from '../utils/GlossaryHelper'
+import StringUtils from '../utils/StringUtils'
+import DateHelpers from '../utils/DateHelper'
+import _ from 'lodash';
+import GlossaryAPIStore from './GlossaryAPIStore'
+
+var PAGE_SIZE = 20,
+  CHANGE_EVENT = "change",
+  MAX_LISTENERS = 100; //number of listener for GlossaryStore (default is 11)
```

Comment:
Says default is 11, but the number here is 100. Is it overriding the default of 11 with this value of 100?